### PR TITLE
Small qalculate tweaks/fixes

### PR DESCRIPTION
**Pull request by XDjackieXD**
Setting the precision in the calculator instance doesn't seem to work so we have to set it in the mathstructure too.
The other thing is the approximation symbol in front of an approximate result (mimics behaviour of many scientific calculators and makes it more obvious that you are dealing with approximations. this is especially useful if the precision is set to a pretty low number).

### CODE REVIEW

File: qalculate/src/extension.cpp [hunk 1]
```diff
@@ -123,9 +123,11 @@ void Qalculate::Extension::handleQuery(Core::Query * query) const {
 
     QString result, error, cmd = query->string().trimmed();
     MathStructure mathStructure;
+
     try {
         string expr = d->calculator->unlocalizeExpression(query->string().toStdString(), d->eo.parse_options);
         mathStructure = d->calculator->calculate(expr, d->eo);
+        mathStructure.setPrecision(d->calculator->getPrecision());
```
Comment (ManuelSchneid3r) [inline qalculate/src/extension.cpp]: why is this needed?

Comment (XDjackieXD) [reply qalculate/src/extension.cpp]: As the main precision setting doesn't seem to work correctly thus using extremely large precision (just try to type in "pi")

Comment (ManuelSchneid3r) [reply qalculate/src/extension.cpp]: whats the version you are using? seems like most OS are still with 0.9

Comment (XDjackieXD) [reply qalculate/src/extension.cpp]: I got version 2.3.0 (latest stable) installed. I primarily use Arch Linux so you can consider everything I do as "tested with latest stable versions of every library" ^^

File: qalculate/src/extension.cpp [hunk 2]
```diff
@@ -138,11 +140,13 @@ void Qalculate::Extension::handleQuery(Core::Query * query) const {
         result = QString::fromStdString(mathStructure.print(d->po));
         auto item = make_shared<StandardItem>(Plugin::id());
         item->setIconPath(d->iconPath);
-        item->setText(result);
-        if ( mathStructure.isApproximate() )
+        if ( mathStructure.isApproximate() ) {
             item->setSubtext(QString("Approximate result of '%1'").arg(cmd));
-        else
+            result.prepend("≈");
```
[the remark this answers is not in the document]
Comment (XDjackieXD) [reply qalculate/src/extension.cpp]: is it possible to prepend something to the output but not copying it when you copy-paste the result?
